docs/resources/networking_subnet_v2.md

```diff
@@ -57,7 +57,7 @@ The following arguments are supported:
 
 * `dns_nameservers` - (Optional) An array of DNS name server names used by hosts
   in this subnet. Changing this updates the DNS name servers for the existing
-  subnet. Default value is `["100.125.4.25", "1.1.1.1"]`
+  subnet. Default value is `["100.125.4.25", "100.125.129.199"]`
```

Comment:
Where  did you take info about 100.125.129.199?

Reply:
I work for the OTC :)

also, when creating a private DNS Zone:

![image](https://user-images.githubusercontent.com/15871966/195939681-2eefd2e7-2ce3-4d53-9ea6-6569ddc5693a.png)